### PR TITLE
EVA-3414 add Writable config script

### PR DESCRIPTION
**Pull request by nitin-ebi**

### CODE REVIEW

File: ebi_eva_common_pyutils/WritableConfig.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        self.content[item] = value
+
+
+def load_config(*args):
+    """Load a config file from any path provided.
+    If none are provided then read from a file path provided in the environment variable CONFIG."""
+    cfg.load_config_file(
+        *args,
+        os.getenv('CONFIG')
+    )
```
Comment (tcezard) [inline ebi_eva_common_pyutils/WritableConfig.py:89]: I don't think this part of [submission_config](https://github.com/EBIvariation/eva-submission/blob/master/eva_submission/submission_config.py) should be lifted to pyutils.

File: ebi_eva_common_pyutils/WritableConfig.py
Comment (tcezard) [inline ebi_eva_common_pyutils/WritableConfig.py:1]: With the way pyutils is organised at the moment, I would prefer to have this class in `config.py`

File: ebi_eva_common_pyutils/WritableConfig.py (hunk 1 above)
Comment (nitin-ebi) [reply ebi_eva_common_pyutils/WritableConfig.py]: removed

File: ebi_eva_common_pyutils/WritableConfig.py
Comment (nitin-ebi) [reply ebi_eva_common_pyutils/WritableConfig.py:1]: moved to config.py

**Review by sundarvenkata-EBI — APPROVED** [review]

**Review by tcezard — APPROVED** [review]

**Pull request merged**